molgenis-platform-integration-tests/src/test/java/org/molgenis/integrationtest/platform/PlatformITConfig.java

```diff
@@ -81,9 +80,9 @@
 		org.molgenis.data.RepositoryCollectionBootstrapper.class, org.molgenis.data.EntityFactoryRegistrar.class,
 		org.molgenis.data.importer.emx.EmxImportService.class, org.molgenis.data.importer.ImportServiceFactory.class,
 		org.molgenis.data.FileRepositoryCollectionFactory.class, org.molgenis.data.excel.ExcelDataConfig.class,
-		org.molgenis.security.permission.PermissionSystemService.class,
```

Comment:
Why is this change needed ? , It is not really clean to me what the '@Import' annotation is needed for here.